.github/workflows/android.yml

```diff
@@ -46,9 +51,39 @@ jobs:
         # Build LLM Demo for Android
         bash build/build_android_llm_demo.sh ${ARTIFACTS_DIR_NAME}
 
+        shasum -a 256 "${ARTIFACTS_DIR_NAME}/llm_demo/executorch.aar"
+
+  upload-release-aar:
+    name: upload-release-aar
+    needs: build-aar
+    if: ${{ startsWith(github.ref, 'refs/tags/android/release') }}
```

Comment:
Just a note that if you use workflow_dispatch, you could run this step when the workflow event_type is workflow_dispatch.  Let me find a concrete example

Reply:
Ah, it's `${{ github.event_name == 'workflow_dispatch' }}`

Reply:
Thank you Huy! 

I am trying with https://github.com/pytorch/executorch/pull/5606 for that path